### ISSUE TITLE
Accessibility improvements on the UI

### ISSUE THREAD

**Issue by gaurav5430** [argoproj/argo-workflows#8807]
# Summary

The UI is lacks accessibility considerations for a lot of UI controls. 
- no proper focus states
- keyboard navigation does not work as expected
- color contrast issues

# Use Cases
- This is required for day to day working

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.
